Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM openjdk:8
+
+ENV SCALA_VERSION=2.12.4 \
+    SBT_VERSION=1.0.3
+
+# Scala expects this file.
+RUN touch /usr/lib/jvm/java-8-openjdk-amd64/release
```

Comment:
this is... odd

Reply:
I found it odd also. I borrowed that from another Dockerfile and at one point in my testing the container broke without it. Just tested again and looks like we don't need it. I'll remove it.